frontend/src/TaskDetail.svelte

```diff
@@ -180,15 +181,17 @@
       if(parts.length == 2) {
         const range = parts[1].split('-');
 
-        selectedRows = null;
+        selectedRows = {
+          path: parts[0],
+          from: parseInt(range[0]),
+          to: parseInt(range[1] || range[0]),
+        };
+
+        let el = document.querySelector('linecode[data-line=' + selectedRows.from + ']');
```

Comment:
is it working for multiple files?

Reply:
we can use this:
```
const el = document.querySelector(`table[data-path="${CSS.escape(parts[0])}"] .linecode[data-line="${CSS.escape(selectedRows.from)"]`);
```

but we need to add `data-path` to table in `SubmitSource.svelte`

Reply:
we should also put this `el` inside `setTimeout` because elements dont exist yet